### PR TITLE
Release invisibility spells

### PR DESCRIPTION
**Pull request by jandersson**
This adds support for releasing invisiblity spells, addressing the issue of attempting to use the bank while invisible, rummaging in containers, and dismantling boxes (which breaks pick.lic)

### CODE REVIEW

File: common-arcana.lic [hunk 1]
```diff
@@ -149,6 +149,15 @@ module DRCA
     result == 'absorbs all of the energy'
   end
 
+  def release_invisibility
```
Comment (jandersson) [inline common-arcana.lic]: This could potentially be combined with `release_cyclics` into `release_spells(property)` or `release_spells(*property)`

**Comment by rpherbig** [comment]
I think this helper should be called in `hunting-buddy`, line 60.

**Comment by rpherbig** [comment]
Note to self: this should fix #945

**Comment by jandersson** [comment]
Closing and reopening for profiles -> data  improvement

**Pull request closed without merging**